test/README.md

```diff
@@ -0,0 +1,94 @@
+# WiredTiger Testing
+
+## Introduction
+The reliability and robustness of WiredTiger is achieved in part by thorough and careful testing.
+
+- C tests: 26939 SLOC
+- CPP tests: 30 SLOC
+- Python tests: 83992 SLOC
+
+## Testing types
+There are many testing areas covered by WiredTiger's test infrasture. Some of the high level areas are:
+
+### Compatibility testing
+> TODO
+
+### Correctness testing
+> TODO
+
+- suite (Python / Unit tests)
+- csuite
+
+### Memory sanitised testing
+> TODO
+
+### Performance testing
+> TODO
+
+- wtperf
+- workgen
+
+### Sanitised testing
+> TODO
+
+### Stress testing
+> TODO
+
+- format
+
+## Test harnesses
```

Comment:
Instead of test harnesses, lets go with test frameworks. 
Under that should be:
- csuite,
- cppsuite,
- workgen,
- wtperf,
- format,
- python.